source/collections/FloatPoint2DList.ooc

```diff
@@ -0,0 +1,93 @@
+/*
+* Copyright (C) 2014 - Simon Mika <simon@mika.se>
+*
+* This sofware is free software; you can redistribute it and/or
+* modify it under the terms of the GNU Lesser General Public
+* License as published by the Free Software Foundation; either
+* version 2.1 of the License, or (at your option) any later version.
+*
+* This software is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+* Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public License
+* along with this software. If not, see <http://www.gnu.org/licenses/>.
+*/
+import VectorList
+import FloatVector
+use ooc-math
+
+FloatPoint2DList: class extends VectorList<FloatPoint2D> {
+	init: func ~default {
+		this super()
+	}
+	init: func ~fromVectorList (other: VectorList<FloatPoint2D>) {
+		this super(other _vector)
+		this count = other count
```

Comment:
I will write this _count. I copied these parts from FloatVector. So it's also inconsistent there.